op-acceptor/runner/runner.go

```diff
@@ -212,7 +214,11 @@ func (r *runner) groupValidatorsByGate() map[string][]types.ValidatorMetadata {
 }
 
 // processGate handles the execution of a single gate and its tests
-func (r *runner) processGate(gateName string, validators []types.ValidatorMetadata, result *RunnerResult) error {
+func (r *runner) processGate(ctx context.Context, gateName string, validators []types.ValidatorMetadata, result *RunnerResult) error {
+	tracer := otel.Tracer("runner")
```

Comment:
`op-acceptor/runner` maybe?

Reply:
I mean maybe, but:
- the tracer name is already nested under the service name (which is "op-acceptor")
- it's seldom used anyway

That said, I definitely shouldn't be creating multiple tracers with the same name. Will fix that

Reply:
Aaah okay so it's nested